src/test/resources/JUISSCSUser.csv

```diff
@@ -0,0 +1,12 @@
+SSCSUserName,SSCSUsrPwd
+juijudgeloadtest1@gmail.com,Password12
+juijudgeloadtest2@gmail.com,Password12
+juijudgeloadtest3@gmail.com,Password12
+juijudgeloadtest4@gmail.com,Password12
+juijudgeloadtest5@gmail.com,Password12
+juijudgeloadtest6@gmail.com,Password12
+juijudgeloadtest7@gmail.com,Password12
+juijudgeloadtest8@gmail.com,Password12
+juijudgeloadtest9@gmail.com,Password12
+juijudgeloadtest10@gmail.com,Password12
+juijudgeloadtest11@gmail.com,Password12
```

Comment:
CR missing here

Reply:
That's correct. Don't want to add a CR in this css otherwise during run time it will pick null username and null password.